docs/mkdocs/reference/lineapy/exceptions/index.md

```diff
@@ -0,0 +1 @@
+::: lineapy.exceptions
```

Comment:
This is apparently the canonical setup documented here.

https://mkdocstrings.github.io/recipes/ 

There are index.md files generated for all directories within `docs/mkdocs/reference` and they are generated from the  `__init__.py` files.  The reason why these main pages are blank is because the `__init__.py` files are empty and have no comments describing the modules they are in.